node-runner-cli/templates/radix-fullnode-compose.yml.j2

```diff
@@ -6,13 +6,27 @@
 version: '3.8'
 services:
 {% if core_node is not none and core_node is defined %}
+  {% if download-community-snapshot == "true" %}
+  download-community-ledger-snapshot:
```

Comment:
What container is this running?

Reply:
My idea is to create a basic ubuntu container to execute  `downloadLedgerSnapshot.sh` script

Reply:
You can potentially use our node container with command arguments.  Just to avoid building maintaining another container?